cpp/tests/test_abm_location.cpp

```diff
@@ -298,12 +303,12 @@ TEST(TestLocation, storeSubpopulations)
         .WillRepeatedly(testing::Return(1.0));
```

Comment:
Please change RecoveredInfected in line before to Recovered

Reply:
Fixed.